source/educators/how-tos/copy_paste_units.rst

```diff
@@ -0,0 +1,77 @@
+****************************
+
+#. Choose a unit that you would like to copy.
+#. In the "actions" hamburger in the upper right corner of the unit, click "Copy
+   to Clipboard".
+
+   ..  image:: /_images/educator_how_tos/copy_paste_outline_page_copy_button.png
+	:alt: The "Copy to Clipboard" button is shown from the hamburger menu on a Unit
+
+#. A notification will appear in the bottom right-hand side of the screen
+   indicating the copy is complete. It should only take a few seconds, depending
+   on the size of the unit.
+
+#. Navigate to the subsection (in the course outline) where you'd like to paste
+   the unit. This may be in the same course or a different course.
+
+#. Click "Paste Unit". The copied unit will appear as the next unit within the
+   subsection. (Note: The "Paste Unit" button will not be visible unless you
+   have copied a unit per step 2.)
+
+   ..  image:: /_images/educator_how_tos/copy_paste_outline_page_paste_button.png
+	:alt: The paste button appears in the outline page within a subsection, below the "New Unit" button
+
+
+From the Unit Page
+******************
+
+#. Choose the unit that you would like to copy from the top horizontal
+   navigation bar.
+
+   ..  image:: /_images/educator_how_tos/copy_paste_unit_page_nav.png
+	:alt: In Studio, the unit editing page is shown, which displays all the units in a subsection in a horizontal navigation bar.
+
+#. Click the "Copy Unit" button from the right-hand sidebar widget.
+
+   ..  image:: /_images/educator_how_tos/copy_paste_unit_page_copy_button.png
+	:alt: The "Copy Unit" button appears in the right sidebar, below the "Publish" button
+
+#. A notification will appear in the bottom right-hand side of the screen
+   indicating the copy is complete. It should only take a few seconds, depending
+   on the size of the unit.
+
+#. Navigate to the subsection where you'd like to paste the unit. This may be in
+   the same course or a different course.
+
+#. In the horizontal navigation bar, click on the dropdown arrow next to the
+   "+New Unit" button. 
+
+#. Click "Paste as new unit".
+
+   ..  image:: /_images/educator_how_tos/copy_paste_unit_page_paste_as_new_unit.png
+	:alt: The "Paste as new unit" button appears in the "+New Unit" dropdown menu
+
+#. Click "Paste Unit". The copied unit will appear as the newest unit within the
+   subsection.
+
+   ..  image:: /_images/educator_how_tos/copy_paste_unit_page_paste_unit.png
+	:alt: The "Paste Unit" button in the horizontal nav bar is shown
+
+#. If you'd like to reorder the newly pasted unit in the subsection, navigate
+   back to the Course Outline page and move the unit.
+
+   ..  image:: /_images/educator_how_tos/course_outline_drag_to_reorder_unit.png
+	:alt: Units can be reordered within a subsection by visiting the Course Outline page and dragging the unit to its desired location
+
+
+.. note::
+   Authors can also paste Units copied from the Unit Page into the Course Outline, and vice versa.
```

Comment:
Looks good, it might be worth having a See-also section with links to copy-pasting content, which someone who is looking at this might also be interested in.

Example code for `seealso`: https://docs.openedx.org/en/latest/documentors/references/doc_templates.html#create-a-how-to